CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ To display last successful and unsuccessful login information, Seacat Auth servi
 
 ### Refactoring
 
+- Change visibility of username to readonly and focus on password input field after wrong password. (INDIGO Sprint 230623, [!35](https://github.com/TeskaLabs/seacat-auth-webui/pull/35))
```

Comment:
@queuing4oranges Put this out of the already released version to release candidate pleas